pandas/tests/series/methods/test_replace.py

```diff
@@ -450,6 +450,18 @@ def test_replace_with_compiled_regex(self):
         expected = pd.Series(["z", "b", "c"])
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize("value", [pd.Period("2020-01"), pd.Interval(0, 5)])
+    def test_replace_ea_ignore_float(self, value):
+        """
+        Regression test for corrolary to GH#34871: if series.replace(1.0, 0.0)
+        is called on a Period/Interval Series, the old, faulty behavior
+        is to raise TypeError.
+        """
```

Comment:
this can be a comment, not a docstring, just needs to see `# GH#34871`, ditto above

Reply:
actually you can share this test using the frame_or_series fixture

Reply:
Changed to a comment, but I grepped for frame_or_series and not finding much in the codebase. I did find `index_or_series`. Can you please elaborate more on how to use frame_or_series? There do seem to be a fair amount of duplication between these two files (series/test_replace and frame/test_replace). `test_replace_with_compiled_regex` is basically copy-pasted between the two, for example

Reply:
> Can you please elaborate more on how to use frame_or_series? 

It's a pytest fixture:
https://github.com/pandas-dev/pandas/blob/aa390ecc825464953f91e9214ed65ef5f83be5d6/pandas/conftest.py#L296

Reply:
Thanks @arw2019  - I needed to merge master in locally to find this